### ISSUE TITLE
Broadcast heart rate (and perhaps other info) to other apps

### ISSUE THREAD

**Issue by rgh36167** [Freeyourgadget/Gadgetbridge#378]
If Gadgetbridge could broadcast its data to other apps like eg https://github.com/jonasoreland/runnerup that would elegantly solve a few other issues like improving the graphs, recording the data and
* https://github.com/Freeyourgadget/Gadgetbridge/issues/377
* https://github.com/Freeyourgadget/Gadgetbridge/issues/372

See  https://github.com/jonasoreland/runnerup/issues/430

**Comment by cpfeiffer** [comment]
Yes, makes sense. I will have a look at that and see if runnerup provides a means to send data to it. Will do so after master is released and we have some progress on Mi2.

**Comment by rgh36167** [comment]
Apparently gadgetbridge already uses broadcasts to send heart rate and realtime steps internally  - maybe allowing other apps to receive those broadcasts (and not terminating them when live mode is not in the foreground) would be a great and fairly simple steps to allow other apps experimenting?

**Comment by Avamander** [comment]
Creating a content provider for other apps to use GadgetBridge's data would be rather nice.

**Comment by cpfeiffer** [comment]
I addd a comment to https://github.com/jonasoreland/runnerup/issues/430 so that we can move forward with this. Sorry that it took us that long.
